spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  let!(:user) { User.create(name: 'Kender', posts_counter: 0) }
+
+  describe 'GET /' do
+    it 'response 200' do
+      get user_posts_path(user)
+
+      expect(response).to have_http_status(200)
+    end
+
+    it 'index template was rendered' do
+      get user_posts_path(user)
+
+      expect(response).to render_template(:index)
+    end
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please make sure to include a test for the `response.body` containing the correct placeholder text. According to the requirements:

![image](https://github.com/quavo19/social-blog/assets/77069011/64f24e44-e2e8-46c3-8884-10f5c23c4ea7)

```rb
# Example
it 'response body includes the correct content' do
  get user_posts_path(user)
  expect(response.body).to match(/posts index/)
end
```


Please consider including this! 🤓 🔧